### ISSUE TITLE
Night shift not working

### ISSUE THREAD

**Issue by Teufeuleu** [kirainmoe/tongfang-utility#59]
I recently updated from Mojave to Big Sur 11.5.2 with an EFI folder made with your (amazing) tool's last 4.0.4 version.
Since then, Night Shift does not seem to work. I can enable/disable it in the Control Center, but there is no color change at all. Also I don't have the Night Shift tab in the Monitor preference panel.
I think it's because the monitor is recognized as "unknown" (even though I can change resolution, refresh rate and brightness), but not sure.
Any idea?

I'm using the GK5CN6Z with the 144Hz monitor.

**Comment by kirainmoe** [comment]
Do you mean that night shift worked in the previous EFI version, and is broken on the latest EFI version? Theoretically I did nothing about the monitor.

**Comment by Teufeuleu** [comment]
Yes but my previous version was quite outdated, I think I made it at the time of TongFang Utility v3.1.4 or so.

**Comment by kirainmoe** [comment]
Please upload both previous version and current version here if convenient.

**Comment by Teufeuleu** [comment]
Unfortunately I deleted backups of the old EFI too soon... [Here's the link](https://file.io/Nv41OXydQaDL) to the new one if it can help tho. I cleaned PlatformInfo data.

**Comment by kirainmoe** [comment]
umm... Maybe something is broken in utility v4. I will investigate this problem.

**Comment by kirainmoe** [comment]
I can not figure out what the problem is. Can you have a try that downgrading WhateverGreen.kext to a previous version?

**Comment by Teufeuleu** [comment]
I tried WG 1.4.8 from march 2021 and same result.
If I get some time I can try to download an old version of the TongFang utility (v3.1.4 for example) and rebuild the EFI from this time and see if the problem is still there. Maybe the problem come from the update I did from Mojave to Big Sur that broke something... I should try to make a clean re-install of Big Sur, as I have near to nothing installed on it.

EDIT: made a clean re-install of Big Sur, but the Night shift problem is the same : no Night Shift tab under Monitor in the pref pan, and Night shift toggle in Control Center not working.
